configs/setups/awiesm/awiesm-2.1.yaml

```diff
@@ -75,6 +83,11 @@ echam:
         namelist_changes:
                 namelist.echam:
                         runctl:
+                                # NOTE(PG): Almost no one in paleodyn uses the
+                                # stream-style output, and most people expect a
```

Comment:
And in fact, even the comment is wrong. We all use the stream-style output, just that the paleodyn workflow expects different streams than are defined by default in echam. I have begun an overall of the namelists for our department, if you are interested, please see here:

https://gitlab.awi.de/paleodyn/Models/namelists

Reply:
That's great, could this repo be syncronised or included within echam namelists inside `esm_tools` GitHub? Then things could be completely automatic.

Reply:
We can discuss that...we could include it as a submodule for example, but git submodules are one level above the git knowledge of some of our users...